examples/spark-s3-to-postgres/whirl.setup.d/00_install_airflow_providers.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+echo "========================================"
+echo "== Install required airflow providers =="
+echo "========================================"
+
+pip install apache-airflow-providers-apache-spark
+aws s3api put-object --bucket ${DEMO_BUCKET} --key input/data/demo/spark/$(date "+%Y%m%d")/data.json --body /mock-data/input.json
```

Comment:
Isn;t this already present in 01_add_mockdata?

Reply:
good catch 👍 Some fat-finger copy-pasting went on there 😉